test/test_torch.py

```diff
@@ -10298,13 +10298,14 @@ def check_single_nuclear_norm(x, axes):
     @skipCUDAIfNoMagma
     def test_nuclear_norm_exceptions(self, device):
         for lst in [], [1], [1, 2]:
-            for axes in (), (0,), (0, 1):
-                x = torch.tensor(lst, dtype=torch.double, device=device)
+            x = torch.tensor(lst, dtype=torch.double, device=device)
+            for axes in (), (0,):
                 self.assertRaises(RuntimeError, torch.norm, x, "nuc", axes)
+            self.assertRaises(IndexError, torch.norm, x, "nuc", (0, 1))
```

Comment:
I had to make this change after adding wrapped dim support to `nuclear_norm`. The `maybe_wrap_dims` call that I added is now where this invalid dimension argument is getting detected and raised. If it's important that we don't change the error type and message, let me know.

Reply:
Changing the error type is technically BC-breaking (and we can add a BC-breaking note to the PR summary describing the change) but it's fine. Changing the message is also fine.